docs/output.md

```diff
@@ -328,7 +328,7 @@ This report includes information on how many reads per sample passed each pipeli
 
 * `pipeline_info/`
     * Reports generated by Nextflow: `execution_report.html`, `execution_timeline.html`, `execution_trace.txt` and `pipeline_dag.dot`/`pipeline_dag.svg`.
-    * Reports generated by the pipeline: `pipeline_report.html`, `pipeline_report.txt` and `software_versions.csv`.
+    * Reports generated by the pipeline: `pipeline_report.html`, `pipeline_report.txt` and `software_versions.yml`. The `pipeline_report*` files will only be present if the `--email` / `--email_on_fail` parameter's are used when running the pipeline.
```

Comment:
Is this true? Why should they only be created when run with an email address?

Reply:
That seems to be true, when testing the pipeline with `-profile test` locally, i.e. without using `--email`, these report files are not generated. I _guess_ `pipeline_report.html` is the file used for generating the report email.

Reply:
I can't see the rationale behind that -- they seem very useful in any case -- but maybe @drpatelh does?